pkg/server/routes.go

```diff
@@ -53,10 +53,8 @@ func (s *Server) routes(
 		s.Config.GetString("CEDAR_API_KEY"),
 	)
 
-	if s.environment.Prod() {
```

Comment:
Previously, these CEDAR checks were only run in the deployed environments: https://github.com/CMSgov/easi-app/commit/b552a5cef14b93ba34ace11aeb9ddb6ce7b088ec. Should we revert back to that?

If we run these checks in ALL environments, that would include local development and CI testing environments.

Reply:
Hrm... Yah. 

Honestly, I kinda hate that `environment` construct, but this is neither the time nor place for me to wage that war. ;-)

I'll go back to `Deployed()`, as a minimally-impacting change.